### PR TITLE
feat: self-hosting docs

### PR DESCRIPTION
**Pull request by ABCxFF**
- Allow support for external postgres
- Fix relative paths in opengb build

### CODE REVIEW

File: src/utils/db.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 const DEFAULT_DATABASE_URL = "postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable";
 
+export function isExternalDatabase(): boolean {
+	return typeof Deno.env.get("DATABASE_URL") === "string";
```
Comment (NathanFlurry) [inline src/utils/db.ts:4]: deno.env.has

File: src/utils/postgres_daemon.ts [hunk 2]
```diff
@@ -13,11 +14,15 @@ const POSTGRES_ONCE = createOnce<void>();
  */
 export async function ensurePostgresRunning(project: Project) {
 	return await getOrInitOnce(POSTGRES_ONCE, async () => {
-		await ensurePostgresRunningInner(project);
+		if (isExternalDatabase()) {
```
Comment (NathanFlurry) [inline src/utils/postgres_daemon.ts:17]: move outside of await, it's faster

File: src/utils/db.ts [hunk 3]
```diff
@@ -1,5 +1,9 @@
 const DEFAULT_DATABASE_URL = "postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable";
 
+export function isExternalDatabase(): boolean {
```
Comment (NathanFlurry) [inline src/utils/db.ts:3]: add brief docs on what this represents and how external db works

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
also needs docs on the database_url under opengb docs /concepts

**Comment by ABCxFF** [comment]
* **#323** <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb/323?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ABCxFF and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Pull request closed without merging**